server/models/user.js

```diff
@@ -20,11 +20,21 @@ const User = new Schema({
     required: true,
     index: { unique: false },
   },
+  terms: {
+    type: [String],
+    required: false,
+  },
+  email: {
+    type: String,
+    required: false,
+  },
+  crawlers: {
+    type: [String],
```

Comment:
Are these basically the specific crawlers that are active? In that case, let's add a enum of potential crawlers that can be added (i.e. twitter, reddit, etc.)

Reply:
Will be handled in the CrawlerService later.
A method that expose available crawlers will be used to sanitize incoming data through a custom validator (see https://mongoosejs.com/docs/validation.html#custom-validators)